sale_promotion_domain_product_discount/models/coupon_reward.py

```diff
@@ -0,0 +1,32 @@
+from odoo import _, fields, models
+
+
+class CouponReward(models.Model):
+    _inherit = "coupon.reward"
+
+    discount_apply_on = fields.Selection(
+        selection_add=[("domain_product", "On Domain Matching Products")],
+    )
+
+    def name_get(self):
+        """
```

Comment:
I've seen this in many places but to me it sounds weird to override the docstring of a method w/ something that is a comment explaining the override.